python/orca/src/bigdl/orca/common.py

```diff
@@ -250,6 +250,10 @@ def init_orca_context(cluster_mode=None, runtime="spark", cores=2, memory="2g",
                 sc = init_spark_on_local(cores, **spark_args)
             elif cluster_mode == "spark-submit":
                 from bigdl.dllib.nncontext import init_nncontext
+                if "conf" in spark_args and spark_args["conf"] is not None:
+                    warnings.warn("For Spark-submit mode, conf should be None, but got "
```

Comment:
Spark-submit mode -> spark-submit cluster_mode

Reply:
change to `all conf should be specified in the spark-submit command`

Reply:
fixed in new commit